### DISCUSSION TITLE
add manualy photos

### DISCUSSION THREAD

**Discussion by constaas** [alicevision/Meshroom#2100] [General]
Good evening everyone, I'm trying to perform photogrammetry of a warehouse to obtain the x, y, z coordinates of 12 specific points inside it. I have taken close-up shots of these points to ensure that the software processes them correctly. Is there a way to manually add these points and directly specify their locations to the software? Because being close-up shots, the software doesn't automatically detect them. Thank you for your help. :)

**Comment by natowi** [comment]
Sorry, this is not possible at the moment. If you use cctags [or 16h5 april tags (linux only)] you can use this method: https://github.com/alicevision/Meshroom/wiki/CCTAG-scaling
